packages/phrases/src/locales/zh-cn.ts

```diff
@@ -608,9 +608,13 @@ const errors = {
     general: '连接器发生未知错误。',
     not_found: '找不到可用的 {{type}} 类型的连接器。',
     not_enabled: '连接器尚未启用。',
+    insufficient_request_parameters: '请求参数缺失。',
     invalid_config: '连接器配置错误。',
+    invalid_response: '连接器错误响应。',
     template_not_found: '无法从连接器配置中找到对应的模板。',
+    auth_code_invalid: '当前连接器的授权码无效。',
```

Comment:
better align with the naming convention above?

`auth_code_invalid` → `invalid_auth_code`